jwst/jump/tests/test_detect_jumps.py

```diff
@@ -16,6 +18,53 @@
 NO_GAIN_VALUE = dqflags.pixel["NO_GAIN_VALUE"]
 
 
+def test_exec_time_0_crs(setup_inputs):
+    """"
+    Set up with dimension similar to simulated MIRI datasets, Dataset has no
+    Ecosmic rays. Test only the execution time of run_detect_jumps() for
+    comparison with nominal time; hopefully indicative of faults with newly
+    added code.
+    """
+    model, rnoise, gain = setup_inputs(ngroups=10, nrows=1024, ncols=1032,
+                                       nints=2, readnoise=6.5, gain=5.5,
+                                       grouptime=2.775, deltatime=2.775)
+
+    tstart = time.time()
+    # using dummy variable in next to prevent "F841-variable is assigned to but never used"
+    _ = run_detect_jumps(model, gain, rnoise, 4.0, 5.0, 6.0, 1, 200, 4, True)
+    tstop = time.time()
+
+    t_elapsed = tstop - tstart
+    MAX_TIME = 10  # takes 1.6 sec on my Mac
+
+    assert t_elapsed < MAX_TIME
+
+
+def test_exec_time_many_crs(setup_inputs):
+    """"
+    Set up with dimension similar to simulated MIRI datasets, Dataset has
+    many cosmic rays; approximately one CR per 4 groups. Test only the execution
+    time of run_detect_jumps() for comparison with nominal time; hopefully
+    indicative of faults with newly added code.
+    """
+    model, rnoise, gain = setup_inputs(ngroups=10, nrows=1024, ncols=1032,
+                                       nints=2, readnoise=6.5, gain=5.5,
+                                       grouptime=2.775, deltatime=2.775)
+
+    crs_frac = 0.25  # fraction of groups having a CR, 726 secs on my Mac
+    model = add_crs(model, crs_frac)  # add desired fraction of CRs
+
+    tstart = time.time()
+    # using dummy variable in next to prevent "F841-variable is assigned to but never used"
+    _ = run_detect_jumps(model, gain, rnoise, 4.0, 5.0, 6.0, 1, 200, 4, True)
+    tstop = time.time()
+
+    t_elapsed = tstop - tstart
+    MAX_TIME = 3000  # takes ~600 sec on my Mac for this crs_frac; takes 1792 in CI test
```

Comment:
This is too long for a unit test. Perhaps reduce the the image size (nrows, ncols).